app/javascript/components/users/SignInComponent.tsx

```diff
@@ -57,47 +78,186 @@ function SignInComponent(props:SignInComponentProps) : JSX.Element {
 		}
 	}, [isValid, componentState]);
 
+
+	//Error messages
 	const { formatMessage } = useIntl();
+	const label = formatMessage({ id: 'email', defaultMessage: '* Requiered' });
+
+
+	//Yup validation
+	const validationSchema = yup.object({
+		email: yup.string().email().required(),
+		password: yup.string().required(),
+	});
+
+	//Styling
+	const useStyles = makeStyles((theme: Theme) => createStyles({
+		textField: {
+			'& .MuiTextField-root': {
+				margin: theme.spacing(1),
+			},
+		},
+		paper: {
+			maxWidth: 400,
+			margin: `${theme.spacing(1)}px auto`,
```

Comment:
Is this the proper way to center something horizontally in Material UI? I'm literally asking, I don't know :smile:

Reply:
Any update here?

Reply:
Any update on this?

Reply:
Hi, I haven't heard back here.

Reply:
We have set maxWidth to 400 because we don't want the paper to get too big when the screen is larger. This allows the paper to only scale until 400 regardless of the screen size. We did make sure the paper looks good on devises with different sizes on the debugger.